src/features/layout/nav/NavBar.tsx

```diff
@@ -0,0 +1,33 @@
+import { NavLink } from 'react-router-dom';
+
+const NavBar = (prop: {
+  t: any;
+  className: string;
+  handleNav?: () => void;
+}) => {
+  return (
+    <>
+      {[
```

Comment:
You could define this array outside of the return statement for cleaner code

Reply:
Done.